Coming_Next.md

```diff
@@ -9,6 +9,7 @@ Here’s the stuff we’re working on next for you and IBM Watson Work Services.
 
 | Coming Next   |
 | :-------------|
+| **Templates.** We're adding **Templates** for you to create custom experiences for teams. It's `EXPERIMENTAL` but you can check it out here, [Space Template](https://github.com/watsonwork/watsonwork-developer-docs/blob/master/guides/V1_space_template_main.md). We have [Create a Space Template](https://github.com/watsonwork/watsonwork-developer-docs/blob/master/guides/V1_create_space_template.md), [Space Template Info](https://github.com/watsonwork/watsonwork-developer-docs/blob/master/guides/V1_space_template_info.md), [Get Space Template](https://github.com/watsonwork/watsonwork-developer-docs/blob/master/guides/V1_get_space_template.md), [Get My Space Template](https://github.com/watsonwork/watsonwork-developer-docs/blob/master/guides/V1_get_my_space_templates.md) and [Delete a Space Template](https://github.com/watsonwork/watsonwork-developer-docs/blob/master/guides/V1_delete_space_template.md). For templates associated with specific offerings, we have [Get Offering Space Templates](https://github.com/watsonwork/watsonwork-developer-docs/blob/master/guides/V1_get_offering_space_templates.md).              |
```

Comment:
One minor thing.... can we pluralize this?  It returns a collection not a template....  it is the analog to getting a list of all the apps you have created.... but for templates...  

>Get My Space Template

to

>Get My Space Templates